lib/css/modules/_showImages.scss

```diff
@@ -285,22 +170,152 @@ div.madeVisible {
 			content: '\F178';
 		}
 	}
-}
 
-.res-player {
-	position: absolute;
+	.res-media-rotatable & {
+		opacity: 0;
+	}
 
-	// Hide "lookup" button on video's
-	.RESMediaControls .imageLookup {
+	.res-media-rotatable:hover & {
+		opacity: .8;
+		transition: opacity 300ms;
+	}
+
+	.res-media-load-error ~ & {
 		display: none;
 	}
 }
 
-// video-advanced
+.res-expando {
+	display: block;
+
+	.usertext-body {
+		user-select: text; // Allow selection if is a text expando
+	}
+
+	.res-expando-media {
+		display: inline-block;
+
+		.usertext-body &,
+		.selftext.expanded ~ * & {
+			max-width: 100%;
+		}
+	}
+
+	.res-expando-independent {
+		z-index: 1;
+		position: absolute;
+	}
+
+	&[hidden],
+	[hidden] {
+		display: none !important;
+	}
+
+	.res-title {
+		font-size: 13px;
+	}
+
+	.res-caption {
+		font-size: 11px;
+		white-space: pre-wrap;
+	}
+
+	.res-credits {
+		font-size: 10px;
+	}
+}
+
+// Override native expando-class, since it restricts the size of inner expandos
+.expando {
+	position: initial !important;
+}
+
+.res-gallery {
+	.res-gallery-title,
+	.res-gallery-caption {
+		max-width: 700px;
+	}
+
+	.res-gallery-title {
+		margin: 3px;
+		font-size: 15px;
+	}
+
+	.res-gallery-caption {
+		max-width: 700px;
+		margin-bottom: 3px;
+		font-size: 12px;
+	}
+
+	.res-gallery-individual-controls {
+		display: flex;
+		align-items: center;
+		margin: 4px 0 4px 10px;
+
+		&[last-piece=true] .res-gallery-next::after,
+		&[first-piece=true] .res-gallery-prev::after {
```

Comment:
This class is `res-gallery-prev` in some places and `res-gallery-previous` in others